plugin/table_fetch.go

```diff
@@ -165,8 +173,15 @@ func (t *Table) getForEach(ctx context.Context, queryData *QueryData, rd *RowDat
 			}()
 			// create a context with the matrix item
 			fetchContext := context.WithValue(ctx, context_key.MatrixItem, matrixItem)
-
-			item, err := getCall(fetchContext, queryData, &HydrateData{})
+			retryConfig := t.Get.RetryConfig
```

Comment:
this is identical to the code above - can we put into a function and share

Reply:
Done here
https://github.com/turbot/steampipe-plugin-sdk/pull/41/files#diff-26d86950248c57ec612c24ae0e84ea49b265f4a75c02c88186b4dbc2c8ec0643R429